lib/des/cipher.js

```diff
@@ -11,6 +11,7 @@ function Cipher(options) {
 
   this.buffer = new Array(this.blockSize);
   this.bufferOff = 0;
+  this.padding = options.padding ?? true
```

Comment:
Could we change this to:
```suggestion
  this.padding = options.padding !== false
```
I'm afraid of breaking compatibility with ancient node.js versions.

Thanks!